src/main.rs

```diff
@@ -29,12 +32,20 @@ static ERRORS: AtomicUsize = AtomicUsize::new(0);
 static WARNINGS: AtomicUsize = AtomicUsize::new(0);
 
 const HELP: &str = concat!(
-    env!("CARGO_PKG_NAME"), " ", env!("RCC_GIT_REV"), "\n",
+    env!("CARGO_PKG_NAME"),
+    " ",
+    env!("RCC_GIT_REV"),
+    "\n",
```

Comment:
This makes it look less like the output. Before one line of source corresponded to one line of help output.

Reply:
I think rustfmt just did that. A little odd